napari_hub_cli/checklist/metadata.py

```diff
@@ -236,12 +241,14 @@ def display_checklist(analysis_result):
         style="bold underline2 blue",
     )
 
-    # Display additional informations
-    for feature in analysis_result.additionals:
-        console.print(f"  {feature.meta.name}: {feature.result}")
-
     # Display summary result
+    previous_title = None
     for feature in analysis_result.features:
+        if feature.meta.section:
+            if feature.meta.section.title != previous_title:
```

Comment:
@SimaoBolota-MetaCell, can you compact those two lines in only one `if` with and `and`?